packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -115,6 +115,13 @@ enum RoutePopDisposition {
   bubble,
 }
 
+/// A placeholder for a route.
+@visibleForTesting
+class RoutePlaceholder {
+  /// A const constructor for [RoutePlaceholder].
```

Comment:
const constructors have standard boilerplate documentation

Reply:
const constructors doc is different across Flutter

may be style guide should be updated with the standard boilerplate?